tests/golden_html/claude-basic.html

```
@@ -0,0 +1,329 @@
+        font-size: 0.85rem;
+        color: var(--accent);
+        text-decoration: none;
+        opacity: 0.7;
+      }
+      .anchor-btn:hover { opacity: 1; }
+    </style>
+  </head>
+  <body>
+    <header>
+      <div class="pill">Golden Claude</div>
+      <input class="search-input" id="client-search" type="search" placeholder="Filter text (press / to focus)" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)